### PR TITLE
[xxx-status-index] proposed draft

### PR DESCRIPTION
**Pull request by yenda**

### CODE REVIEW

File: ideas/000-status-index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    Idea: 000-status-index
+    Title: Status Index
+    Status: draft
+    Created: 2018-04-27
+
+## Summary
+
+We have a growing number of very useful standalone webpages that are providing valuable
+informations and services. Exemples of those are cluster visualization, cluster status,
```
Comment (PombeirP) [inline ideas/000-status-index.md]: typo: `Examples`

File: ideas/000-status-index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+history of the #status channel on whisper, documentation...
+
+The goal of this swarm is to build an index of these pages so that they can be listed
+and easely accessed from a single place. We also want to make sure that each of these
```
Comment (PombeirP) [inline ideas/000-status-index.md]: easily

File: ideas/000-status-index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    Idea: 000-status-index
+    Title: Status Index
+    Status: draft
+    Created: 2018-04-27
+
+## Summary
+
+We have a growing number of very useful standalone webpages that are providing valuable
+informations and services. Exemples of those are cluster visualization, cluster status,
+history of the #status channel on whisper, documentation...
```
Comment (PombeirP) [inline ideas/000-status-index.md]: maybe add '#status channel on whisper *on the various networks*'

File: ideas/000-status-index.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+The goal of this swarm is to build an index of these pages so that they can be listed
```
[the remark this answers is not in the document]
Comment (yenda) [reply ideas/000-status-index.md]: You mentionned him on Slack I hope he'll join in.

File: ideas/225-status-index.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
+---
+id: 225-status-index
+title: Status Index
+status: draft
```
Comment (oskarth) [inline ideas/225-status-index.md]: Draft

File: ideas/225-status-index.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+Description:
+
+## Success Metrics
+
+> Assuming the idea ships, what would success look like? What are the most
+> important metrics that you would move?
+>
+> Example: Onboarding conversion rate. Target >30% full funnel
+
+## Exit criteria
```
Comment (oskarth) [inline ideas/225-status-index.md]: It'd be useful if we can figure out exit criteria and success metrics, this way the problem/swarm is bounded in terms of what the most important thing it is trying to do is. Also to figure out the overlap with existing documentation efforts, etc.

**Comment by oskarth** [comment]
Merging as draft.

**Pull request merged**